mdz/pkg/cmd/deploy.go

```diff
@@ -56,10 +58,37 @@ func init() {
 	deployCmd.Flags().IntVar(&deployGPU, "gpu", 0, "Number of GPUs")
 	deployCmd.Flags().StringVar(&deployName, "name", "", "Name of inference")
 	deployCmd.Flags().StringSliceVarP(&deployNodeLabel, "node-labels", "l", []string{}, "Node labels")
+	deployCmd.Flags().BoolVar(&deployDetach, "detach", false, "If set, the command returns immediately without waiting for the deployment to complete")
 	deployCmd.Flags().StringVar(&deployCommand, "command", "", "Command to run")
 	deployCmd.Flags().StringVar(&deployProbePath, "probe-path", "", "HTTP Health probe path")
 }
 
+func waitForDeploymentReady(cmd *cobra.Command, client *client.Client, namespace, name string, interval time.Duration, timeoutSeconds int) error {
```

Comment:
How come the `interval` is duration while `timeoutSeconds` is int? The parameter type should be consistent.